classes/MainMenuTab.cpp

```diff
@@ -0,0 +1,33 @@
+#include "MainMenuTab.h"
+#include "LayerHandler.h"
+#include "Button.h"
+
+MainMenuTab::MainMenuTab(Game* game) : Renderable(OVERLAY), mBackground_sprite(nullptr), mGame(game)
+{
+}
+
+void MainMenuTab::render()
+{
+	SDL_Rect dest;
+
+	dest.x = 0;
+	dest.y = 0;
+	dest.w = 1920;
+	dest.h = 1080;
```

Comment:
Kannst du auch in einer Zeile konstruieren, und bitte statt 1920 und 1080 die LOGICAL_SCREEN_WIDTH\HEIGHT verwenden
```suggestion
	SDL_Rect dest{0, 0, LOGICAL_SCREEN_WIDTH, LOGICAL_SCREEN_HEIGHT};
```